pom.xml

```diff
@@ -54,25 +49,19 @@
     </repository>
   </repositories>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-kafka-client</artifactId>
-        <version>${quarkus.version}</version>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <dependencies>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-kafka-client</artifactId>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Seems this is still junit4, is this still needed?

Reply:
Actually no need. Just adding it as the following error, but Im not sure why it reports this. Or if we can handle it on parent pom. 
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test (default-test) on project indy-event-model: groups/excludedGroups require TestNG or JUnit48+ on project test classpath -> [Help 1]`

Reply:
This is weird, not sure why this surefire plugin is 2.18.1 here. This plugin has been set to 2.22.2 in parent pom, which does not need junit4 anymore.

Reply:
Okay, I will remove it here. thanks.